### PR TITLE
Revisit 990

### PR DESCRIPTION
**Pull request by jorg3lopez**
# Remove Sender and Receiver entities

This PR removes the sender and receiver columns from the database and from our code base. The sender and receiver can be access through the `sendingFacilityDetails` and the `receivingFacilityDetails`. Due to this, the isolated sender and receiver become redundant code.

The main work was to remove these entities from the `PartnertMetadata` class and from the database.

## Issue
#990 

## Checklist

- [x] I have added tests to cover my changes

### CODE REVIEW

File: etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/etor/metadata/partner/PartnerMetadataOrchestratorTest.groovy [hunk 1]
```diff
@@ -26,7 +26,6 @@ class PartnerMetadataOrchestratorTest extends Specification {
     def hashCode = "hash"
     def bearerToken = "token"
     def sender = "sender"
```
Comment (basiliskus) [inline etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/etor/metadata/partner/PartnerMetadataOrchestratorTest.groovy:28]: I think this `sender` can be removed

File: examples/Test/e2e/orders/002_ORM_O01.fhir [hunk 2]
```diff
@@ -1,1406 +1,1626 @@
 {
```
Comment (basiliskus) [inline examples/Test/e2e/orders/002_ORM_O01.fhir:1]: Why was this file changed? We need to preserve some files in the examples folder as other tests may rely on it (like the load tests). In the case of this particular one, I think we want to preserve it because it was based on an example we got from MN. If necessary, it's better to add a new file and point to that one

**Review by basiliskus — COMMENTED** [review]
Left a couple of comments, otherwise looks good to me!

File: examples/Test/e2e/orders/002_ORM_O01.fhir (hunk 2 above)
Comment (halprin) [reply examples/Test/e2e/orders/002_ORM_O01.fhir:1]: This FHIR file indeed came from an MN HL7v2 file, but it was outdated based on the old RS HL7v2/FHIR conversion algorithm.  For example, nothing was coming out as the sender and receiver because this FHIR file was missing how RS now extracts that information.  So, we basically reran the original HL7v2 file through RS again to get the updated FHIR file.

File: etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/etor/metadata/partner/PartnerMetadataOrchestratorTest.groovy (hunk 1 above)
Comment (jorg3lopez) [reply etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/etor/metadata/partner/PartnerMetadataOrchestratorTest.groovy]: Great catch!

**Review by basiliskus — APPROVED** [review]

**Review by halprin — APPROVED** [review]
Ship it.
